Makefile.am

```diff
@@ -691,14 +691,14 @@ EXTRA_DIST += dist/tpm-udev.rules
 install-dirs:
 if HOSTOS_LINUX
 if SYSD_SYSUSERS
-	systemd-sysusers || echo "*** WARNING *** Failed to create the tss user and group"
+	systemd-sysusers $(DESTDIR)$(sysconfdir)/sysusers.d/tpm2-tss.conf
```

Comment:
This is going to make the installation fail if `systemd-sysusers` cannot be successfully executed, e.g. because the installation is not done as root. Please use
```
-systemd-sysusers $(DESTDIR)$(sysconfdir)/sysusers.d/tpm2-tss.conf
```
or
```
systemd-sysusers $(DESTDIR)$(sysconfdir)/sysusers.d/tpm2-tss.conf || true
```
to make the installation continue in this case, see #2022.

As a side note, adding the configuration file to `systemd-sysusers` or `systemd-tmpfiles` is not really necessary, if left out, all configuration files in the default paths will be considered, but specifying our own config file shouldn't hurt either.